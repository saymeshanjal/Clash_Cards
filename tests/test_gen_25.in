11
16 367 2
16 246 5
16 319 5
4 84 3
10 27 3
2 127 2
12 380 1
8 286 2
10 294 5
6 269 1
6 188 5
