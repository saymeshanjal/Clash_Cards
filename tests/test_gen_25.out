4081 4082 4707
4248 8622 0
4248 8622 0
1 5 0
68 184 0
0 1 1
462 462 0
20 21 29
45 207 0
10 10 0
1 19 0
